source/BoardingPanel.h

```diff
@@ -108,6 +107,7 @@ class BoardingPanel : public Panel {
 		int64_t unitValue;
 		std::string size;
 		std::string value;
+		bool knownToPlayer;
```

Comment:
```suggestion
		bool knownToPlayer = false;
```
Add a dedault otherwise if you dont define it somewhere for some reason and try to access it it'll crash